website/src/content/docs/docs/howtos/ARM/versioning.md

```diff
@@ -362,3 +362,60 @@ model EmployeeProperties {
   city?: string;
 }
 ```
+
+## FAQ
```

Comment:
Its tagged at the end of the arm versioning doc but as we don't have a generic/data plane one this feels fine